index.html

```diff
@@ -3,6 +3,7 @@
   <head>
     <!-- <script type="module" src="index.js"></script> -->
     <script type="module" src="https://unpkg.com/hls-video-element@0"></script>
+    <script type="module" src="https://unpkg.com/@mux/mux-video@0/dist/mux-video.mjs"></script>
```

Comment:
could this be changed to jsDelivr?

Reply:
Yes, I just kept it as unpkg as that's what the other URL was.

Reply:
Updated.